readthedocs/organizations/templates/organizations/organization_choose.html

```diff
@@ -0,0 +1,51 @@
+{% extends "base.html" %}
+
+{% load i18n %}
+{% load gravatar %}
+{% load organizations %}
+
+{% block title %}{% trans "Before continuing, choose an organization..." %}{% endblock %}
+
+{% block organization-bar-organization %}active{% endblock %}
+
+{% block content %}
+  <div class="col-major organization-major">
+    <div class="module organizations organizations-list">
+      <div class="module-wrapper">
+
+        <h2>{% trans "Before continuing, choose an organization..." %}</h2>
+
+        <p>
+          {% trans "You are a member of several organizations." %}<br>
+          {% trans "Click the organization that you want to use:" %}
```

Comment:
A few notes here:

- This should be a single, trimmed blocktrans block
- I'd drop the additional `<br>`, it looks unnecessary in display
- Also "click" -> "select"

Reply:
![image](https://github.com/readthedocs/readthedocs.org/assets/374612/39ecb129-9868-4eca-b0ab-70d339739ba2)

Reply:
I was unsure if you wanted to also shorten the headline here?

Reply:
Yeah, noted above that the title and heading here could both be shortened.

Reply:
Alright, done